dockerfiles/units/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ruby:3.3
```

Comment:
Do we really need a separate dockerfile to run unit-tests?
This will result increase of maintenance cost

Maybe we can just use default Dockerfile and just replace `CMD` on runtime inside docker actions?
docker engine is supporting it will be something

```
docker build . --name doc-builder
docker run -it docbuilder rake unit_test
```

Reply:
We planned to do this next PR, but I think it would be even better to do it right away.